src/DI/CzechPostExtension.php

```diff
@@ -27,10 +29,26 @@ class CzechPostExtension extends CompilerExtension
 		],
 	];
 
+	public function getConfigSchema(): Nette\Schema\Schema
+	{
+		return Expect::structure(
+			[
+				'http' => Expect::structure([
+					'base_uri' => Expect::string('https://online.postservis.cz/'),
+					'auth' => Expect::array(['', '']),
+														 ]),
+				'config' => Expect::structure([
+					 'tmp_dir' => Expect::type('string|null')->default(null),
+														 ]),
```

Comment:
Formatting is broken.

Reply:
I don't see problem, tried run codestyle checker `make cs` and for sure `make csf` without error.

Reply:
<img width="802" alt="czech-post_CzechPostExtension_php_at_fd47c566c5db1559d832c8d05517c00ebb39803c_·_contributte_czech-post" src="https://user-images.githubusercontent.com/538058/86099461-cb406d00-bab7-11ea-9091-a6c6c3b902ce.png">

Reply:
Hope, now it's ok.